plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/model/aws/chat/ChatMessage.kt

```diff
@@ -4,7 +4,7 @@
 package software.aws.toolkits.jetbrains.services.amazonq.lsp.model.aws.chat
 
 data class ChatMessage(
-    val type: MessageType? = MessageType.ANSWER,
+    val type: String? = MessageType.ANSWER.value,
```

Comment:
?

Reply:
for some reason MessageType enum is not correctly parsed and will always return null when receiving from flare, change it to string so that the correct type is returned to correctly render history ask/answer texts.

Reply:
https://github.com/aws/mynah-ui/blob/ba21eba86d0e867583449bf76cf41dbc57f570dc/src/static.ts#L245-L254

our type definition is not complete

Reply:
well I guess yes,
but OpenTab is using https://github.com/aws/language-server-runtimes/blob/256c549df6d446886457dd83ea4b71f31d3bf063/types/chat.ts#L136, that has only 5 types, 
so at least JB aligns with it and I feel this is enough now.

Reply:
my understanding is that we are just passing this and send it to browser, what it contains doesn't matter; we can remove maintaining this enum